platform/security/platform-security-core-api/src/main/java/ddf/security/SubjectUtils.java

```diff
@@ -193,4 +199,35 @@ public static String getEmailAddress(Subject subject) {
                 .findFirst()
                 .orElse(null);
     }
+
+    public static Set<String> getRoles(Subject subject) {
```

Comment:
Should we consider making this a getAttribute method and it takes the attribute name like the role attribute URI instead of adding a helper for roles?